ingest/indexer/service/indexer_streaming_service.go

```diff
@@ -240,6 +245,43 @@ func (s *indexerStreamingService) publishTxn(ctx context.Context, req types.Requ
 	return s.client.PublishTransaction(sdkCtx, txn)
 }
 
+// addTokenLiquidity adds the token liquidity to the event.
+// It refers to the pooled amount of each asset after a swap event has occurred.
+func (s *indexerStreamingService) AddTokenLiquidity(ctx context.Context, event *types.Event) error {
+	if event.Type != gammtypes.TypeEvtTokenSwapped {
+		return nil
+	}
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
+	var poolIdStr string
+	attributes := event.Attributes
+	// Find the pool id from the token_swapped
+	for _, attribute := range attributes {
+		if attribute.Key == concentratedliquiditytypes.AttributeKeyPoolId {
+			poolIdStr = attribute.Value
+			break
+		}
+	}
+	if poolIdStr == "" {
+		return errors.New("pool id not found")
+	}
+	poolId, err := strconv.ParseInt(poolIdStr, 10, 64)
+	if err != nil {
+		return err
+	}
+	coins, err := s.keepers.PoolManagerKeeper.GetTotalPoolLiquidity(sdkCtx, uint64(poolId))
```

Comment:
nit: Is there a way to parse to uint directly?

Reply:
Got it 👍 good one 👍